docsource/modules130-140.rst

```diff
@@ -218,7 +218,7 @@ Module coverage 13.0 -> 14.0
 +--------------------------------------------+-------------------------------------------------+
 |l10n_ar                                     |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-| |new| l10n_ar_website_sale                 |                                                 |
+| |new| l10n_ar_website_sale                 | Done                                            |
```

Comment:
@MiquelRForgeFlow Why mark this module as complete? What about the module `website_sale`?

Reply:
Good catch, I didn't see that.